backend/infrahub/core/protocols.py

```diff
@@ -482,6 +482,7 @@ class CoreProposedChange(CoreTaskTarget):
     destination_branch: String
     state: Enum
     is_draft: Boolean
+    total_comments: IntegerOptional
```

Comment:
I'm wondering if this will be problematic in some way later. I'd prefer to have this as an Integer instead of optional. The read_only attribute could have a default value of 0. However I'm not certain if the schema upgrade will handle that. It could be that we should revisit this part later. Not a high priority it's just nicer to know that an empty set would always return a `0` which I think it would but according to the schema it would be valid with `0 | None`. Given the time if we need an extra migration for this we can perhaps create a housekeeping issue to change it later.